src/dmd/staticcond.d

```diff
@@ -93,6 +94,8 @@ bool evalStaticCondition(Scope* sc, Expression exp, Expression e, ref bool error
         return false;
     }
 
+    e = resolveAliasThis(sc,e);
```

Comment:
Mind the style: `e = resolveAliasThis(sc, e);`

Reply:
Why not use dfmt or as part of precommit hook?

Reply:
It's a good idea in general, I'm not sure if dfmt is prosuction ready for enforcing dmd's code style. This seems like a good experiment to try if you have time.

Reply:
I tried this a while ago to get dfmt integrated with phobos. But I stumbled upon an edge case which I couldn't figure out, so I dropped it. Let me try it again.